internal/pkg/config/service.go

```diff
@@ -25,29 +25,29 @@ const (
 	AddonsCfnTemplateNameFormat = "%s.addons.stack.yml"
 )
 
-// Service represents a deployable long running service or task.
-type Service struct {
-	App  string `json:"app"`  // Name of the app this service belongs to.
-	Name string `json:"name"` // Name of the service, which must be unique within a app.
-	Type string `json:"type"` // Type of the service (ex: Load Balanced Web Server, etc)
+// Workload represents a deployable long running service or task.
+type Workload struct {
+	App  string `json:"app"`  // Name of the app this workload belongs to.
+	Name string `json:"name"` // Name of the workload, which must be unique within a app.
+	Type string `json:"type"` // Type of the workload (ex: Load Balanced Web Server, etc)
 }
 
-// CreateService instantiates a new service within an existing application. Skip if
-// the service already exists in the application.
-func (s *Store) CreateService(svc *Service) error {
-	if _, err := s.GetApplication(svc.App); err != nil {
+// CreateWorkload instantiates a new service or job within an existing application. Skip if
+// the workload already exists in the application.
+func (s *Store) CreateWorkload(wkld *Workload) error {
+	if _, err := s.GetApplication(wkld.App); err != nil {
 		return err
 	}
 
-	servicePath := fmt.Sprintf(fmtSvcParamPath, svc.App, svc.Name)
-	data, err := marshal(svc)
+	servicePath := fmt.Sprintf(fmtSvcParamPath, wkld.App, wkld.Name)
+	data, err := marshal(wkld)
 	if err != nil {
-		return fmt.Errorf("serializing service %s: %w", svc.Name, err)
+		return fmt.Errorf("serializing workload %s: %w", wkld.Name, err)
 	}
 
 	_, err = s.ssmClient.PutParameter(&ssm.PutParameterInput{
 		Name:        aws.String(servicePath),
```

Comment:
can we rename the const to `workloadPath`?

Reply:
✅